app/components/toolbar/toolbar.html

```diff
@@ -1,15 +1,12 @@
-<div class="toolbar" ng-class="{'toolbar--active': toolbarActive}">
-    <div class="toolbar__logo">
-    </div>
-
-    <div class="toolbar__content">
-        Live Management
-    </div>
-    <a class="toolbar__close" ng-click="toggleToolbar()">
-        <i class="icon-close-thick"></i>
-    </a>
-
-    <a class="toolbar__open" ng-click="toggleToolbar()">
-    </a>
-
+<div class="leToolbar" ng-class="{'leToolbar--active': toolbarActive}">

```

Comment:
the toolbar is transparent when there is more text and when you scroll down, see below:

<img width="1204" alt="zrzut ekranu 2017-01-25 15 58 35" src="https://cloud.githubusercontent.com/assets/562536/22295467/57847e7c-e317-11e6-9d98-1996e71e70d8.png">